Specifications/Language/1_ProgramStructure/3_CallableDeclarations.md

```diff
@@ -40,4 +40,10 @@ Here, `body` specifies that the given implementation applies to the default body
 
 The same applies to operations; when there is exactly one specialization defining the implementation of the default body, the additional wrapping of the form `body (...){ <implementation> }` may be omitted.
 
+## Recursion
+
+Q# callables can be directly or indirectly recursive and can be declared in any order; an operation or function may call itself, or it may call another callable that directly or indirectly calls the caller. 
```

Comment:
This is indeed explained in the section on type parameterization. I think there is no additional link needed, since that is specific to type parameterized things and that section is already linked where I mention that type parameterized callables exist.